### PR TITLE
fix cn docs for dice_loss, log_loss, unfold and NLLLoss API 

### PR DESCRIPTION
**Pull request by lijianshe02**
update cn docs for dice_loss, log_loss, unfold and nll_loss API
dice_zh
log_zh
nll_zh
unfold_zh

### CODE REVIEW

File: doc/paddle/api/paddle/fluid/layers/dice_loss_cn.rst [hunk 1]
```diff
@@ -31,7 +31,6 @@ dice_loss定义为:
 
     import paddle
     import paddle.nn.functional as F
-    paddle.disable_static()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/dice_loss_cn.rst:34]: 第六行可以改一下 改成 paddle.nn.functionl. dice_loss

File: doc/paddle/api/paddle/fluid/layers/log_loss_cn.rst [hunk 2]
```diff
@@ -34,7 +34,6 @@ log_loss
     import paddle
     import paddle.nn.functional as F
     
-    paddle.disable_static()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/log_loss_cn.rst:37]: 第六行可以改为 paddle.nn.functional.log_loss

File: doc/paddle/api/paddle/fluid/layers/unfold_cn.rst [hunk 3]
```diff
@@ -50,15 +50,17 @@ unfold
 
 返回：   unfold操作之后的结果，形状如上面所描述的[N, Cout, Lout]，Cout每一个滑动block里面覆盖的元素个数，Lout是滑动block的个数，数据类型与 ``x`` 相同
 
-返回类型：    Variable
+返回类型：    Tensor
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/unfold_cn.rst]: 返回类型删掉，返回改为：
返回：Tensor，unfold操作之后的结果，形状如上面所....

File: doc/paddle/api/paddle/fluid/layers/unfold_cn.rst [hunk 4]
```diff
@@ -40,7 +40,7 @@ unfold
 
 
 参数：
-    - **x**  (Variable) – 输入4-D Tensor，形状为[N, C, H, W]，数据类型为float32或者float64
+    - **x**  (Tensor) – 输入4-D Tensor，形状为[N, C, H, W]，数据类型为float32或者float64
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/unfold_cn.rst:43]: 第六行也可以改为 paddle.nn.functional.unfold

File: doc/paddle/api/paddle/nn/layer/loss/NLLLoss_cn.rst [hunk 5]
```diff
@@ -61,8 +61,6 @@ NLLLoss
                              [0.05689114, 0.0862954 , 0.6325046 ]]).astype(np.float32)
         label_np = np.array([0, 2, 1, 1, 0]).astype(np.int64)
 
-        place = paddle.CPUPlace()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/layer/loss/NLLLoss_cn.rst:64]: 第六行改为 paddle.nn.NLLLoss
54行同理

File: doc/paddle/api/paddle/nn/functional/loss/nll_loss_cn.rst [hunk 6]
```diff
@@ -36,8 +36,6 @@ nll_loss
                          [0.05689114, 0.0862954 , 0.6325046 ]]).astype(np.float32)
         label_np = np.array([0, 2, 1, 1, 0]).astype(np.int64)
 
-        place = paddle.CPUPlace()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/nll_loss_cn.rst:39]: 43行 print(result) 即可 不需要 print(result.numpy()) 其他部分也是一样的~

Comment (lijianshe02) [reply doc/paddle/api/paddle/nn/functional/loss/nll_loss_cn.rst:39]: 好的，谢谢

File: doc/paddle/api/paddle/fluid/layers/dice_loss_cn.rst (hunk 1 above)
Comment (lijianshe02) [reply doc/paddle/api/paddle/fluid/layers/dice_loss_cn.rst:34]: 好的，已改

File: doc/paddle/api/paddle/fluid/layers/log_loss_cn.rst (hunk 2 above)
Comment (lijianshe02) [reply doc/paddle/api/paddle/fluid/layers/log_loss_cn.rst:37]: 好的，已改

File: doc/paddle/api/paddle/fluid/layers/unfold_cn.rst (hunk 3 above)
Comment (lijianshe02) [reply doc/paddle/api/paddle/fluid/layers/unfold_cn.rst]: 好的，已改

File: doc/paddle/api/paddle/nn/layer/loss/NLLLoss_cn.rst (hunk 5 above)
Comment (lijianshe02) [reply doc/paddle/api/paddle/nn/layer/loss/NLLLoss_cn.rst:64]: 好的，已改

**Review by TCChenlong — APPROVED** [review]
LGTM
